webpack.common.js

```diff
@@ -1,95 +1,133 @@
 const webpack = require('webpack')
 const path = require('path')
+const glob = require('glob')
 const CopyWebpackPlugin = require('copy-webpack-plugin')
 const MiniCssExtractPlugin = require('mini-css-extract-plugin')
 
 module.exports = {
   entry: {
-    adhocracy4: [
-      './digitalstrategie/assets/scss/style.scss',
-      './digitalstrategie/assets/js/app.js'
-    ],
-    vendor: [
+    digitalstrategie: [ // array of entry points
       '@fortawesome/fontawesome-free/scss/fontawesome.scss',
       '@fortawesome/fontawesome-free/scss/brands.scss',
       '@fortawesome/fontawesome-free/scss/regular.scss',
       '@fortawesome/fontawesome-free/scss/solid.scss',
-      'jquery',
+      './digitalstrategie/assets/scss/style.scss',
+      './digitalstrategie/assets/js/app.js'
     ],
   },
   output: {
-    libraryTarget: 'this',
-    library: '[name]',
+    // exposes exports of entry points
+    library: {
+      name: '[name]',
+      // return value of entry point will be assigned this.
+      type: 'this'
+    },
+    // creates a folder to store all assets
     path: path.resolve('./digitalstrategie/static/'),
-    publicPath: '/static/',
-    filename: '[name].js'
+    // location they can be accessed, can also be a url
+    publicPath: '/static/'
   },
   externals: {
-    'django': 'django'
+    django: 'django'
+  },
+  // enables assets property for loading
+  experiments: {
+    asset: true
   },
   module: {
     rules: [
       {
         test: /\.jsx?$/,
-        exclude: /node_modules\/(?!(bootstrap)\/).*/, // exclude all dependencies but bootstrap
-        loader: 'babel-loader',
-        options: {
-          presets: ['@babel/preset-env',].map(require.resolve),
-          plugins: ['@babel/plugin-transform-runtime', '@babel/plugin-transform-modules-commonjs']
+        exclude: /node_modules\/(?!(bootstrap)\/).*/, // exclude all dependencies but adhocracy4
```

Comment:
do we actually still need this?

Reply:
This is not outdated, I only changed the comment...